### PR TITLE
DCS-1595 WPIP landing page changes

### PR DESCRIPTION
**Pull request by davidthomas22**

### CODE REVIEW

File: integration_tests/pages/recentArrivals/recentArrivals.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export default class RecentArrivalsPage extends Page {
     doNotScan: () => cy.get(`[data-qa=recent-arrival-${date}-${row}] [data-qa=do-not-scan]`),
   })
 
-  noRecentArrivlsOnDay = (date: string): PageElement => cy.get(`[data-qa=no-prisoners-${date}`)
+  noRecentArrivalsOnDay = (date: string): PageElement => cy.get(`[data-qa=no-prisoners-${date}`)
```
Comment (davidthomas22) [inline integration_tests/pages/recentArrivals/recentArrivals.ts:26]: Noticed typo whilst updating tests

File: integration_tests/integration/recentArrivals/recentArrivals.cy.ts [hunk 2]
```diff
@@ -47,9 +47,9 @@ context('A user can view all recent arrivals', () => {
       .should('contain.text', `${moment().format('D MMMM YYYY')}, 14:40`)
     recentArrivalsPage.recentArrivals(1, today).location().should('contain.text', 'MDI-1-3-004')
     recentArrivalsPage.recentArrivals(1, today).doNotScan().should('not.exist')
-    recentArrivalsPage.noRecentArrivlsOnDay(today).should('not.exist')
+    recentArrivalsPage.noRecentArrivalsOnDay(today).should('not.exist')
```
Comment (davidthomas22) [inline integration_tests/integration/recentArrivals/recentArrivals.cy.ts:50]: Noticed typo's whilst updating tests

File: server/views/pages/home.njk [hunk 3]
```diff
@@ -19,28 +19,43 @@
...
-                            "description": 'Confirm the return of prisoners out on temporary absence.',
-                            "data-qa": "return-from-temporary-absence"
+                            "heading": 'People booked to arrive today',
+                            "description": 'Confirm the arrival of prisoners arriving from court, a custody suite or another establishment.',
+                            "data-qa": "choose-prisoner"
                         }) }}
                     </li>
-                {% endif %}
-            </ul>
+                    {% if temporaryAbsencesEnabled %}
```
Comment (GurnankCheema) [inline server/views/pages/home.njk:33]: TEMPORARY_ABSENCE_ENABLED is true across dev, pre and prod so can remove this conditional check across the app.

**Review by GurnankCheema — APPROVED** [review]
LGTM

**Review by GurnankCheema — APPROVED** [review]
LGTM

**Pull request merged**